.travis.yml

```diff
@@ -2,10 +2,11 @@ language: php
 
 dist: trusty
 
-php: [5.4, 5.5, 7.0]
+php: [5.4, 5.5, 5.6, 7.0, 7.1, 7.2, 7.3, 7.4]
 
 before_script:
   - composer self-update
+  - if [[ ${TRAVIS_PHP_VERSION:0:3} =~ ^7\.[234]$ ]] ; then composer require --dev phpunit/phpunit:"^5.7.27" ; fi
```

Comment:
Will this also be an issue for PHP versions > 7.4? Why do we have to use PHPUnit v5 for these versions?

Reply:
It is not a problem of raygun4php, but if using PHPUnit 3.x with PHP7.2 or higher, it will be a deprecated error [like this](https://travis-ci.org/tenkoma/raygun4php/jobs/629309039?utm_medium=notification&utm_source=github_status), so I tried using PHPUnit 5.x.

This problem will also occur with PHP versions higher than 7.4. However, it wasn’t released yet, and it seems that version 2.0 was under development on the master branch, so I thought it was enough until 7.4.